backend/init_data/skills/dingtalk-knowledge/provider.py

```
@@ -0,0 +1,700 @@
+        if self.ws_emitter:
+            try:
+                await self.ws_emitter.emit_tool_call(
+                    task_id=self.task_id,
+                    tool_name=self.name,
+                    tool_input={
+                        "download_url": download_url[:50] + "...",
+                        "filename": filename,
+                    },
+                    status="running",
+                )
+            except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)